generate-docs/scripts/preprocessor.ts

```diff
@@ -124,6 +109,53 @@ tryCatch(async () => {
         dtsBuilder.extractDtsSection(definitions, "Begin Word APIs", "End Word APIs")
     );
 
+    // ----
+    // Process Custom Functions d.ts
+    // ----
+    const dtsForCfs = "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/custom-functions-runtime/index.d.ts";
+    fsx.writeFileSync("../script-inputs/custom-functions-runtime.d.ts", await fetchAndThrowOnError(dtsForCfs, "text"));
+    console.log(`\nReading from ${path.resolve("../script-inputs/custom-functions-runtime.d.ts")}`);
+    let definitionsForCfs : string = fsx.readFileSync("../script-inputs/custom-functions-runtime.d.ts").toString();
+
+    console.log("Fixing issues with d.ts file...");
+    definitionsForCfs = definitionsForCfs.replace(/^(\s*)(declare namespace)(\s+)/gm, `$1export $2$3`)
+        .replace(/^(\s*)(declare module)(\s+)/gm, `$1export $2$3`)
+        .replace(/^(\s*)(namespace)(\s+)/gm, `$1export $2$3`)
+        .replace(/^(\s*)(class)(\s+)/gm, `$1export $2$3`)
+        .replace(/^(\s*)(interface)(\s+)/gm, `$1export $2$3`)
+        .replace(/^(\s*)(module)(\s+)/gm, `$1export $2$3`)
+        .replace(/^(\s*)(function)(\s+)/gm, `$1export $2$3`)
+        .replace(/(\s*)(@param)(\s+)(\w+)(\s)(\s)/g, `$1$2$3$4$5`)
+        .replace(/(\s*)(@param)(\s+)(\w+)(\s+)([^\-])/g, `$1$2$3$4$5- $6`);
+
+    console.log("create file: custom-functions-runtime.d.ts");
+    fsx.writeFileSync('../api-extractor-inputs-custom-functions-runtime/custom-functions-runtime.d.ts', definitionsForCfs);
+
+    // ----
+    // Process Office Runtime d.ts
+    // ----
+    const dtsForORun = "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-runtime/index.d.ts";
```

Comment:
Same comments about local file options.

Reply:
Yep, good suggestion. I've added a prompt about source of ORT d.ts file and updated processing logic accordingly.